src/Helpers/Token.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class TokenHelper {
+    public static function createNewToken($token, $user, $remember = false)
+    {
+        $obj = new stdClass();
+        $obj->access_token = $token;
+        $obj->token_type = 'bearer';
+        $obj->user = $user;
```

Comment:
use camelCase for variable naming. Don't worry about the json response, badaso ApiResponse will handle it for us.
ref: https://xqsit.github.io/laravel-coding-guidelines/docs/naming-conventions/